### ISSUE TITLE
Continuous query does not count according to the set time window

### ISSUE THREAD

**Issue by darrenji** [taosdata/TDengine#2222]
连续查询没有按照设定的time window统计。比如，按连续查询的定义应该每分钟统计一次，但并没有，而是在第3、4分钟，第8、9分钟有值，而在第1，2，5，6，7分钟却没有统计。

> 场景：一台设备向时序数据库每分钟上传一次数据，这台设备下有多条分路。也就是说，上传数据靠这台设备，而时序数据库需要记录的是这多条分路的数据。

具体如下：

设备每分钟上传一次数据，需要存储所有分路的实时数据。首先需要一张超级表：
```
create table superreal (ts timestamp, addr binary(5), val float) tags (deviceNo binary(20));
```

superreal超级表的子表是real_001, real_002......
```
create table real_001 using superreal tags ('001');
create table real_002 using superreal tags ('002');
```

以上，每台设备每分钟的实时数据存放在了对应的子表real_001, real_002......

现在需要每分钟对real_001, real_002......统计一次，使用连续查询对每条分路的数据求和存放到tj_001,tj_002......中。连续查询是这样定义的：
```
create table tj_001 as select sum(val) from real_001 interval(1m);
create table tj_002 as select sum(val) from real_002 interval(1m);
```

拿real_001举例，查询其最近60分钟之内的实时数据：
```
select * from real_001 where ts >= now - 60m;
```

```
          ts          |addr |         val        |
=================================
 20-06-10 10:55:32.453|1    |             3.52000|
 20-06-10 10:55:32.454|2    |            11.45000|
 20-06-10 10:55:32.455|3    |             5.35000|
 20-06-10 10:55:32.456|4    |             1.00000|
 20-06-10 10:55:32.457|5    |             0.00000|
 20-06-10 10:55:32.458|6    |             1.57000|
 20-06-10 10:55:32.459|7    |             3.57000|
 20-06-10 10:56:38.610|1    |             3.52000|
 20-06-10 10:56:38.611|2    |            11.45000|
 20-06-10 10:56:38.612|3    |             5.35000|
 20-06-10 10:56:38.613|4    |             1.00000|
 20-06-10 10:56:38.614|5    |             0.00000|
 20-06-10 10:56:38.615|6    |             1.28000|
 20-06-10 10:56:38.616|7    |             2.92000|
 20-06-10 10:57:39.837|1    |             3.52000|
 20-06-10 10:57:39.838|2    |            11.45000|
 20-06-10 10:57:39.839|3    |             5.35000|
 20-06-10 10:57:39.840|4    |             1.00000|
 20-06-10 10:57:39.841|5    |             0.00000|
 20-06-10 10:57:39.842|6    |             1.20000|
 20-06-10 10:57:39.843|7    |             2.72000|
 20-06-10 10:58:37.451|1    |             3.53000|
 20-06-10 10:58:37.452|2    |            11.47000|
 20-06-10 10:58:37.453|3    |             5.35000|
 20-06-10 10:58:37.454|4    |             1.00000|
 20-06-10 10:58:37.455|5    |             0.00000|
 20-06-10 10:58:37.456|6    |             1.14000|
 20-06-10 10:58:37.457|7    |             2.56000|
 20-06-10 10:59:08.826|1    |             3.52000|
 20-06-10 10:59:08.827|2    |            11.46000|
 20-06-10 10:59:08.828|3    |             5.35000|
 20-06-10 10:59:08.829|4    |             1.00000|
 20-06-10 10:59:08.830|5    |             0.00000|
 20-06-10 10:59:08.831|6    |             1.82000|
 20-06-10 10:59:08.832|7    |             3.96000|
 20-06-10 11:00:36.272|1    |             3.52000|
 20-06-10 11:00:36.273|2    |            11.46000|
 20-06-10 11:00:36.274|3    |             5.35000|
 20-06-10 11:00:36.275|4    |             1.00000|
 20-06-10 11:00:36.276|5    |             0.00000|
 20-06-10 11:00:36.277|6    |             1.16000|
 20-06-10 11:00:36.278|7    |             2.64000|
 20-06-10 11:01:42.633|1    |             3.52000|
 20-06-10 11:01:42.634|2    |            11.46000|
 20-06-10 11:01:42.635|3    |             5.35000|
 20-06-10 11:01:42.636|4    |             1.00000|
 20-06-10 11:01:42.637|5    |             0.00000|
 20-06-10 11:01:42.638|6    |             1.35000|
 20-06-10 11:01:42.639|7    |             3.06000|
 20-06-10 11:02:36.894|1    |             3.52000|
 20-06-10 11:02:36.895|2    |            11.47000|
 20-06-10 11:02:36.896|3    |             5.35000|
 20-06-10 11:02:36.897|4    |             1.00000|
 20-06-10 11:02:36.898|5    |             0.00000|
 20-06-10 11:02:36.899|6    |             1.39000|
 20-06-10 11:02:36.900|7    |             3.14000|
 20-06-10 11:03:40.221|1    |             3.52000|
 20-06-10 11:03:40.222|2    |            11.47000|
 20-06-10 11:03:40.223|3    |             5.35000|
 20-06-10 11:03:40.224|4    |             1.00000|
 20-06-10 11:03:40.225|5    |             0.00000|
 20-06-10 11:03:40.226|6    |             1.13000|
 20-06-10 11:03:40.227|7    |             2.58000|
 20-06-10 11:04:07.777|1    |             3.52000|
 20-06-10 11:04:07.778|2    |            11.47000|
 20-06-10 11:04:07.779|3    |             5.35000|
 20-06-10 11:04:07.780|4    |             1.00000|
 20-06-10 11:04:07.781|5    |             0.00000|
 20-06-10 11:04:07.782|6    |             1.22000|
 20-06-10 11:04:07.783|7    |             2.75000|
 20-06-10 11:05:31.654|1    |             3.52000|
 20-06-10 11:05:31.655|2    |            11.47000|
 20-06-10 11:05:31.656|3    |             5.35000|
 20-06-10 11:05:31.657|4    |             1.00000|
 20-06-10 11:05:31.658|5    |             0.00000|
 20-06-10 11:05:31.659|6    |             1.49000|
 20-06-10 11:05:31.660|7    |             3.40000|
 20-06-10 11:06:42.432|1    |             3.52000|
 20-06-10 11:06:42.433|2    |            11.47000|
 20-06-10 11:06:42.434|3    |             5.35000|
 20-06-10 11:06:42.435|4    |             1.00000|
 20-06-10 11:06:42.436|5    |             0.00000|
 20-06-10 11:06:42.437|6    |             1.57000|
 20-06-10 11:06:42.438|7    |             3.55000|
 20-06-10 11:07:37.514|1    |             3.52000|
 20-06-10 11:07:37.515|2    |            11.47000|
 20-06-10 11:07:37.516|3    |             5.36000|
 20-06-10 11:07:37.517|4    |             1.00000|
 20-06-10 11:07:37.518|5    |             0.00000|
 20-06-10 11:07:37.519|6    |             1.80000|
 20-06-10 11:07:37.520|7    |             4.09000|
 20-06-10 11:08:34.471|1    |             3.52000|
 20-06-10 11:08:34.472|2    |            11.46000|
 20-06-10 11:08:34.473|3    |             5.35000|
 20-06-10 11:08:34.474|4    |             1.00000|
 20-06-10 11:08:34.475|5    |             0.00000|
 20-06-10 11:08:34.476|6    |             1.37000|
 20-06-10 11:08:34.477|7    |             3.08000|
 20-06-10 11:09:08.674|1    |             3.53000|
 20-06-10 11:09:08.675|2    |            11.48000|
 20-06-10 11:09:08.676|3    |             5.36000|
 20-06-10 11:09:08.677|4    |             1.00000|
 20-06-10 11:09:08.678|5    |             0.00000|
 20-06-10 11:09:08.679|6    |             1.15000|
 20-06-10 11:09:08.680|7    |             2.58000|
 20-06-10 11:10:33.046|1    |             3.52000|
 20-06-10 11:10:33.047|2    |            11.46000|
 20-06-10 11:10:33.048|3    |             5.35000|
 20-06-10 11:10:33.049|4    |             1.00000|
 20-06-10 11:10:33.050|5    |             0.00000|
 20-06-10 11:10:33.051|6    |             1.66000|
 20-06-10 11:10:33.052|7    |             3.81000|
 20-06-10 11:11:42.577|1    |             3.52000|
 20-06-10 11:11:42.578|2    |            11.47000|
 20-06-10 11:11:42.579|3    |             5.35000|
 20-06-10 11:11:42.580|4    |             1.00000|
 20-06-10 11:11:42.581|5    |             0.00000|
 20-06-10 11:11:42.582|6    |             1.38000|
 20-06-10 11:11:42.583|7    |             3.02000|
 20-06-10 11:12:35.211|1    |             3.53000|
 20-06-10 11:12:35.212|2    |            11.48000|
 20-06-10 11:12:35.213|3    |             5.36000| 
 20-06-10 11:12:35.214|4    |             1.00000|
 20-06-10 11:12:35.215|5    |             0.00000|
 20-06-10 11:12:35.216|6    |             1.16000| 
 20-06-10 11:12:35.217|7    |             2.62000|
 20-06-10 11:13:36.786|1    |             3.52000|
 20-06-10 11:13:36.787|2    |            11.47000|
 20-06-10 11:13:36.788|3    |             5.35000| 
 20-06-10 11:13:36.789|4    |             1.00000|
 20-06-10 11:13:36.790|5    |             0.00000|
 20-06-10 11:13:36.791|6    |             1.16000|
 20-06-10 11:13:36.792|7    |             2.63000|
 20-06-10 11:14:04.276|1    |             3.52000|
 20-06-10 11:14:04.277|2    |            11.45000|
 20-06-10 11:14:04.278|3    |             5.34000|
 20-06-10 11:14:04.279|4    |             1.00000|
 20-06-10 11:14:04.280|5    |             0.00000|
 20-06-10 11:14:04.281|6    |             1.10000| 
 20-06-10 11:14:04.282|7    |             2.45000|
 20-06-10 11:15:35.531|1    |             3.52000| 
 20-06-10 11:15:35.532|2    |            11.45000|
 20-06-10 11:15:35.533|3    |             5.34000|
 20-06-10 11:15:35.534|4    |             1.00000|
 20-06-10 11:15:35.535|5    |             0.00000|
 20-06-10 11:15:35.536|6    |             1.09000|
 20-06-10 11:15:35.537|7    |             2.47000|
 20-06-10 11:16:43.265|1    |             3.52000|
 20-06-10 11:16:43.266|2    |            11.46000|
 20-06-10 11:16:43.267|3    |             5.35000|
 20-06-10 11:16:43.268|4    |             1.00000|
 20-06-10 11:16:43.269|5    |             0.00000|
 20-06-10 11:16:43.270|6    |             1.29000|
 20-06-10 11:16:43.271|7    |             2.92000|   
 20-06-10 11:17:34.991|1    |             3.52000|     
 20-06-10 11:17:34.992|2    |            11.46000|  
 20-06-10 11:17:34.993|3    |             5.35000| 
 20-06-10 11:17:34.994|4    |             1.00000| 
 20-06-10 11:17:34.995|5    |             0.00000| 
 20-06-10 11:17:34.996|6    |             1.24000|    
 20-06-10 11:17:34.997|7    |             2.79000|   
 20-06-10 11:18:35.162|1    |             3.52000|    
 20-06-10 11:18:35.163|2    |            11.47000|    
 20-06-10 11:18:35.164|3    |             5.35000|   
 20-06-10 11:18:35.165|4    |             1.00000|    
 20-06-10 11:18:35.166|5    |             0.00000|      
 20-06-10 11:18:35.167|6    |             1.51000|       
 20-06-10 11:18:35.168|7    |             3.46000|         
 20-06-10 11:19:07.321|1    |             3.53000|          
 20-06-10 11:19:07.322|2    |            11.47000|           
 20-06-10 11:19:07.323|3    |             5.36000|           
 20-06-10 11:19:07.324|4    |             1.00000|          
 20-06-10 11:19:07.325|5    |             0.00000|           
 20-06-10 11:19:07.326|6    |             1.53000|          
 20-06-10 11:19:07.327|7    |             3.64000|          
 20-06-10 11:20:36.012|1    |             3.52000|          
 20-06-10 11:20:36.013|2    |            11.46000|           
 20-06-10 11:20:36.014|3    |             5.34000|          
 20-06-10 11:20:36.015|4    |             1.00000|          
 20-06-10 11:20:36.016|5    |             0.00000|          
 20-06-10 11:20:36.017|6    |             1.23000|          
 20-06-10 11:20:36.018|7    |             2.79000|          
 20-06-10 11:21:41.016|1    |             3.52000|          
 20-06-10 11:21:41.017|2    |            11.47000|          
 20-06-10 11:21:41.018|3    |             5.35000|          
 20-06-10 11:21:41.019|4    |             1.00000|          
 20-06-10 11:21:41.020|5    |             0.00000|          
 20-06-10 11:21:41.021|6    |             1.40000|          
 20-06-10 11:21:41.022|7    |             3.19000|           
 20-06-10 11:22:37.598|1    |             3.52000|          
 20-06-10 11:22:37.599|2    |            11.46000|           
 20-06-10 11:22:37.600|3    |             5.35000|          
 20-06-10 11:22:37.601|4    |             1.00000|           
 20-06-10 11:22:37.602|5    |             0.00000|           
 20-06-10 11:22:37.603|6    |             1.28000|           
 20-06-10 11:22:37.604|7    |             2.92000|           
 20-06-10 11:23:36.875|1    |             3.53000|          
 20-06-10 11:23:36.876|2    |            11.47000|          
 20-06-10 11:23:36.877|3    |             5.35000|          
 20-06-10 11:23:36.878|4    |             1.00000|          
 20-06-10 11:23:36.879|5    |             0.00000|          
 20-06-10 11:23:36.880|6    |             1.46000|          
 20-06-10 11:23:36.881|7    |             3.33000|          
 20-06-10 11:24:07.572|1    |             3.53000|          
 20-06-10 11:24:07.573|2    |            11.47000|          
 20-06-10 11:24:07.574|3    |             5.36000|           
 20-06-10 11:24:07.575|4    |             1.00000|          
 20-06-10 11:24:07.576|5    |             0.00000|          
 20-06-10 11:24:07.577|6    |             1.73000|           
 20-06-10 11:24:07.578|7    |             3.98000|          
 20-06-10 11:25:34.212|1    |             3.52000|         
 20-06-10 11:25:34.213|2    |            11.46000|          
 20-06-10 11:25:34.214|3    |             5.35000|           
 20-06-10 11:25:34.215|4    |             1.00000|           
 20-06-10 11:25:34.216|5    |             0.00000|           
 20-06-10 11:25:34.217|6    |             1.33000|           
 20-06-10 11:25:34.218|7    |             3.01000|         
 20-06-10 11:26:41.346|1    |             3.53000|         
 20-06-10 11:26:41.347|2    |            11.50000|          
 20-06-10 11:26:41.348|3    |             5.37000|       
 20-06-10 11:26:41.349|4    |             1.00000|       
 20-06-10 11:26:41.350|5    |             0.00000|         
 20-06-10 11:26:41.351|6    |             1.84000|         
 20-06-10 11:26:41.352|7    |             4.23000|        
 20-06-10 11:27:36.501|1    |             3.54000|          
 20-06-10 11:27:36.502|2    |            11.52000|          
 20-06-10 11:27:36.503|3    |             5.38000|           
 20-06-10 11:27:36.504|4    |             1.01000|           
 20-06-10 11:27:36.505|5    |             0.00000|           
 20-06-10 11:27:36.506|6    |             1.57000|          
 20-06-10 11:27:36.507|7    |             3.56000|          
 20-06-10 11:28:34.118|1    |             3.54000|          
 20-06-10 11:28:34.119|2    |            11.52000|          
 20-06-10 11:28:34.120|3    |             5.37000|        
 20-06-10 11:28:34.121|4    |             1.01000|          
 20-06-10 11:28:34.122|5    |             0.00000|           
 20-06-10 11:28:34.123|6    |             1.26000|           
 20-06-10 11:28:34.124|7    |             2.87000|        
 20-06-10 11:29:08.419|1    |             3.54000|          
 20-06-10 11:29:08.420|2    |            11.51000|         
 20-06-10 11:29:08.421|3    |             5.36000|         
 20-06-10 11:29:08.422|4    |             1.00000|         
 20-06-10 11:29:08.423|5    |             0.00000|          
 20-06-10 11:29:08.424|6    |             1.19000|          
 20-06-10 11:29:08.425|7    |             2.69000|           
 20-06-10 11:30:36.705|1    |             3.54000|          
 20-06-10 11:30:36.706|2    |            11.51000|          
 20-06-10 11:30:36.707|3    |             5.37000|        
 20-06-10 11:30:36.708|4    |             1.00000|          
 20-06-10 11:30:36.709|5    |             0.00000|          
 20-06-10 11:30:36.710|6    |             1.59000|          
 20-06-10 11:30:36.711|7    |             3.62000|          
 20-06-10 11:31:45.263|1    |             3.55000|           
 20-06-10 11:31:45.264|2    |            11.55000|           
 20-06-10 11:31:45.265|3    |             5.38000|          
 20-06-10 11:31:45.266|4    |             1.01000|          
 20-06-10 11:31:45.267|5    |             0.00000|          
 20-06-10 11:31:45.268|6    |             1.59000|          
 20-06-10 11:31:45.269|7    |             3.64000|           
 20-06-10 11:32:35.989|1    |             3.55000|           
 20-06-10 11:32:35.990|2    |            11.56000|          
 20-06-10 11:32:35.991|3    |             5.39000|          
 20-06-10 11:32:35.992|4    |             1.00000|          
 20-06-10 11:32:35.993|5    |             0.00000|           
 20-06-10 11:32:35.994|6    |             1.86000|          
 20-06-10 11:32:35.995|7    |             4.24000|          
 20-06-10 11:33:37.851|1    |             3.55000|         
 20-06-10 11:33:37.852|2    |            11.56000|         
 20-06-10 11:33:37.853|3    |             5.39000|        
 20-06-10 11:33:37.854|4    |             1.01000|         
 20-06-10 11:33:37.855|5    |             0.00000|          
 20-06-10 11:33:37.856|6    |             1.77000|          
 20-06-10 11:33:37.857|7    |             4.07000|          
 20-06-10 11:34:08.795|1    |             3.55000|          
 20-06-10 11:34:08.796|2    |            11.56000|           
 20-06-10 11:34:08.797|3    |             5.39000|          
 20-06-10 11:34:08.798|4    |             1.00000|          
 20-06-10 11:34:08.799|5    |             0.00000|         
 20-06-10 11:34:08.800|6    |             1.30000|         
 20-06-10 11:34:08.801|7    |             2.96000|          
 20-06-10 11:35:35.707|1    |             3.55000|         
 20-06-10 11:35:35.708|2    |            11.56000|         
 20-06-10 11:35:35.709|3    |             5.39000|           
 20-06-10 11:35:35.710|4    |             1.01000|        
 20-06-10 11:35:35.711|5    |             0.00000|       
 20-06-10 11:35:35.712|6    |             1.82000| 
 20-06-10 11:35:35.713|7    |             4.20000|         
 20-06-10 11:36:41.935|1    |             3.55000|         
 20-06-10 11:36:41.936|2    |            11.55000|          
 20-06-10 11:36:41.937|3    |             5.38000|          
 20-06-10 11:36:41.938|4    |             1.00000|           
 20-06-10 11:36:41.939|5    |             0.00000|          
 20-06-10 11:36:41.940|6    |             1.31000|         
 20-06-10 11:36:41.941|7    |             2.97000|         
 20-06-10 11:37:37.987|1    |             3.55000|         
 20-06-10 11:37:37.988|2    |            11.55000|          
 20-06-10 11:37:37.989|3    |             5.39000|          
 20-06-10 11:37:37.990|4    |             1.01000|          
 20-06-10 11:37:37.991|5    |             0.00000|           
 20-06-10 11:37:37.992|6    |             1.43000|         
 20-06-10 11:37:37.993|7    |             3.28000|        
 20-06-10 11:38:37.165|1    |             3.55000|         
 20-06-10 11:38:37.166|2    |            11.56000|        
 20-06-10 11:38:37.167|3    |             5.39000|         
 20-06-10 11:38:37.168|4    |             1.01000|         
 20-06-10 11:38:37.169|5    |             0.00000|          
 20-06-10 11:38:37.170|6    |             0.96000|          
 20-06-10 11:38:37.171|7    |             2.16000|          
 20-06-10 11:39:08.409|1    |             3.55000|           
 20-06-10 11:39:08.410|2    |            11.55000|          
 20-06-10 11:39:08.411|3    |             5.37000|           
 20-06-10 11:39:08.412|4    |             1.01000|          
 20-06-10 11:39:08.413|5    |             0.00000|          
 20-06-10 11:39:08.414|6    |             1.07000|          
 20-06-10 11:39:08.415|7    |             2.38000|          
 20-06-10 11:40:31.578|1    |             3.55000|          
 20-06-10 11:40:31.579|2    |            11.54000|          
 20-06-10 11:40:31.580|3    |             5.38000|          
 20-06-10 11:40:31.581|4    |             1.00000|          
 20-06-10 11:40:31.582|5    |             0.00000|          
 20-06-10 11:40:31.583|6    |             1.18000|         
 20-06-10 11:40:31.584|7    |             2.68000|          
 20-06-10 11:41:43.736|1    |             3.55000|           
 20-06-10 11:41:43.737|2    |            11.55000|          
 20-06-10 11:41:43.738|3    |             5.38000|          
 20-06-10 11:41:43.739|4    |             1.01000|          
 20-06-10 11:41:43.740|5    |             0.00000|          
 20-06-10 11:41:43.741|6    |             1.18000|         
 20-06-10 11:41:43.742|7    |             2.68000|      
 20-06-10 11:42:36.922|1    |             3.56000| 
 20-06-10 11:42:36.923|2    |            11.58000|       
 20-06-10 11:42:36.924|3    |             5.39000|      
 20-06-10 11:42:36.925|4    |             1.01000|        
 20-06-10 11:42:36.926|5    |             0.00000|     
 20-06-10 11:42:36.927|6    |             1.50000|        
 20-06-10 11:42:36.928|7    |             3.44000|         
 20-06-10 11:43:37.487|1    |             3.56000|        
 20-06-10 11:43:37.488|2    |            11.58000|          
 20-06-10 11:43:37.489|3    |             5.39000|          
 20-06-10 11:43:37.490|4    |             1.00000|           
 20-06-10 11:43:37.491|5    |             0.00000|          
 20-06-10 11:43:37.492|6    |             1.15000|           
 20-06-10 11:43:37.493|7    |             2.70000|          
 20-06-10 11:44:08.839|1    |             3.56000|           
 20-06-10 11:44:08.840|2    |            11.58000|          
 20-06-10 11:44:08.841|3    |             5.39000|          
 20-06-10 11:44:08.842|4    |             1.01000|          
 20-06-10 11:44:08.843|5    |             0.00000|          
 20-06-10 11:44:08.844|6    |             1.54000|        
 20-06-10 11:44:08.845|7    |             3.49000|          
 20-06-10 11:45:36.251|1    |             3.55000|        
 20-06-10 11:45:36.252|2    |            11.56000|         
 20-06-10 11:45:36.253|3    |             5.39000|         
 20-06-10 11:45:36.254|4    |             1.00000|          
 20-06-10 11:45:36.255|5    |             0.00000|          
 20-06-10 11:45:36.256|6    |             1.61000|          
 20-06-10 11:45:36.257|7    |             3.69000|           
 20-06-10 11:46:43.921|1    |             3.55000|          
 20-06-10 11:46:43.922|2    |            11.55000|         
 20-06-10 11:46:43.923|3    |             5.38000|           
 20-06-10 11:46:43.924|4    |             1.00000|          
 20-06-10 11:46:43.925|5    |             0.00000|           
 20-06-10 11:46:43.926|6    |             1.14000|           
 20-06-10 11:46:43.927|7    |             2.55000|          
 20-06-10 11:47:35.808|1    |             3.55000|          
 20-06-10 11:47:35.809|2    |            11.57000|           
 20-06-10 11:47:35.810|3    |             5.38000|          
 20-06-10 11:47:35.811|4    |             1.01000|          
 20-06-10 11:47:35.812|5    |             0.00000|        
 20-06-10 11:47:35.813|6    |             1.29000|         
 20-06-10 11:47:35.814|7    |             2.94000|          
 20-06-10 11:48:42.188|1    |             3.56000|          
 20-06-10 11:48:42.189|2    |            11.56000|          
 20-06-10 11:48:42.190|3    |             5.39000|          
 20-06-10 11:48:42.191|4    |             1.01000|           
 20-06-10 11:48:42.192|5    |             0.00000|          
 20-06-10 11:48:42.193|6    |             0.94000|          
 20-06-10 11:48:42.194|7    |             2.10000|          
 20-06-10 11:49:07.855|1    |             3.56000|         
 20-06-10 11:49:07.856|2    |            11.56000|         
 20-06-10 11:49:07.857|3    |             5.40000|         
 20-06-10 11:49:07.858|4    |             1.01000|         
 20-06-10 11:49:07.859|5    |             0.00000|           
 20-06-10 11:49:07.860|6    |             1.80000|           
 20-06-10 11:49:07.861|7    |             4.14000|          
 20-06-10 11:50:33.273|1    |             3.55000|          
 20-06-10 11:50:33.274|2    |            11.57000|          
 20-06-10 11:50:33.275|3    |             5.39000|          
 20-06-10 11:50:33.276|4    |             1.00000|          
 20-06-10 11:50:33.277|5    |             0.00000|           
 20-06-10 11:50:33.278|6    |             1.01000|          
 20-06-10 11:50:33.279|7    |             2.62000|          
 20-06-10 11:51:44.665|1    |             3.55000|         
 20-06-10 11:51:44.666|2    |            11.56000|        
 20-06-10 11:51:44.667|3    |             5.39000|          
 20-06-10 11:51:44.668|4    |             1.01000|          
 20-06-10 11:51:44.669|5    |             0.00000|          
 20-06-10 11:51:44.670|6    |             1.11000|          
 20-06-10 11:51:44.671|7    |             2.53000|        
 20-06-10 11:52:35.939|1    |             3.55000|         
 20-06-10 11:52:35.940|2    |            11.56000|          
 20-06-10 11:52:35.941|3    |             5.38000|          
 20-06-10 11:52:35.942|4    |             1.01000|         
 20-06-10 11:52:35.943|5    |             0.00000|          
 20-06-10 11:52:35.944|6    |             1.20000|          
 20-06-10 11:52:35.945|7    |             2.70000|           
 20-06-10 11:54:08.529|1    |             3.55000|          
 20-06-10 11:54:08.530|2    |            11.56000|          
 20-06-10 11:54:08.531|3    |             5.39000|          
 20-06-10 11:54:08.532|4    |             1.01000|          
 20-06-10 11:54:08.533|5    |             0.00000|          
 20-06-10 11:54:08.534|6    |             1.19000|          
 20-06-10 11:54:08.535|7    |             2.69000|         
```

再取出tj_001中最近60分钟之内的统计数据：
```
select * from tj_001 where ts >= now - 60m;
```

```
          ts          |         sum_val_         | 
==============================
 20-06-10 10:58:00.000|             25.050000072|
 20-06-10 10:59:00.000|             27.110000014| 
 20-06-10 11:03:00.000|             25.050000072|
 20-06-10 11:04:00.000|             25.310000181|
 20-06-10 11:08:00.000|             25.779999852|
 20-06-10 11:09:00.000|             25.099999547|
 20-06-10 11:13:00.000|             25.130000234| 
 20-06-10 11:14:00.000|             24.860000014|
 20-06-10 11:18:00.000|             26.310000181|
 20-06-10 11:19:00.000|             26.530000448|
 20-06-10 11:23:00.000|             26.140000105|
 20-06-10 11:24:00.000|             27.070000410|
 20-06-10 11:28:00.000|             25.570000172|
 20-06-10 11:29:00.000|             25.290000439| 
 20-06-10 11:33:00.000|             27.350000381| 
 20-06-10 11:34:00.000|             25.760000229|  
 20-06-10 11:38:00.000|             24.630000293|
 20-06-10 11:39:00.000|             24.930000186| 
 20-06-10 11:43:00.000|             25.379999757| 
 20-06-10 11:44:00.000|             26.569999695| 
 20-06-10 11:48:00.000|             24.560000122|
 20-06-10 11:49:00.000|             27.470000267|
 20-06-10 11:54:00.000|             25.390000343| 
```

最后，问题来了。

在连续查询的定义中：
```
create table tj_001 as select sum(val) from real_001 interval(1m);
```

tj_001应该每分钟出一次统计数据，为什么却没有呢？

**Comment by Aries-Lee1991** [comment]
wait, the R&D will check it.

**Comment by localvar** [comment]
may I know the version number of your TDengine instance? thanks

**Comment by darrenji** [comment]
Community Edition, version is 1.6.6.0

**Comment by localvar** [comment]
we think the root cause of this issue is the delay of data insertion, for example: data with timestamp `10:55:32` is inserted into database at `10:56:00` or later.

as v2.0 is expected to be released soon, we will implement a fix in the new version and leave v1.6 as it is.

but please understand the fix is only a mitigation, because TDengine cannot count a data in if it arrives too late unless making a full recalculation, which is both expensive and hard to implement.
